src/services/apiService.ts

```diff
@@ -64,27 +83,31 @@ export async function getCommits(owner: string, repo: string): Promise<ICommit[]
     }
 
     const dataValue: ICommit = {
-      author: commitData.commit.author,
+      author: commitData.author,
       message: commitData.commit.message,
-      branches: commitData.parents.map(parent => parent.sha),
-      date: commitData.commit.author?.date ? commitData.commit.author.date.substring(0,10) : "Unknown",
+      branches: commitData.parents.map((parent) => parent.sha),
+      date: commitData.commit.author?.date
```

Comment:
A consideration: usually, we want to avoid else if ternary statements as they can be unreadable and a bit difficult to maintain. To achieve that, we can use nullish coalescing in this way: `commitData.commit.author?.date?.substring(0, 10) ?? 'Unknown'`. Nothing wrong with the logic, this is just something to consider.